python/common/org/python/types/Dict.java

```diff
@@ -297,10 +297,15 @@ public void __delitem__(org.python.Object item) {
     }
 
     @org.python.Method(
-        __doc__ = ""
+        __doc__ = "",
+        default_args = {"other", "default_value"}
     )
-    public org.python.Object get(org.python.Object other) {
-        throw new org.python.exceptions.NotImplementedError("dict.get() has not been implemented.");
+    public org.python.Object get(org.python.Object other, org.python.Object default_value) {
+        try {
+            return this.__getitem__(other);
+        } catch (org.python.exceptions.KeyError e){ // allow unhashable type error to be percolated up.
+            return default_value != null ? default_value: org.python.types.NoneType.NONE;
```

Comment:
Ternary if syntax is a recipe for confusion :-) Roll this out as a normal if statement.